test/unit/mods.example.ts

```diff
@@ -1656,6 +1656,51 @@ export const FallbackForNonMatchedAndInvalidShouldPromptForInstall = new Map<
       cancelLabel: InstallChoices.Cancel,
       cancelErrorMessage: expectedUserCancelMessageFor(InstallerType.Fallback),
     },
+    // Fallback for mods containing JSON that will later be handled by AMM
+    validAmmModUsingFallback: {
+      expectedInstallerType: InstallerType.Fallback,
+      inFiles: [
+        ...ARCHIVE_PREFIXES,
+        path.join(ARCHIVE_PREFIX, "proximas_propshop_v4.archive"),
+        ...pathHierarchyFor(
+          path.normalize("bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/"),
+        ),
+        "bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/Collabs/",
+        "bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/Collabs/Custom Props/",
+        "bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/Collabs/Custom Props/proximas_propshop_v4.lua",
+        "bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/User/",
+        "bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/User/Decor/",
+        "bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/User/Decor/Cyber Noir Flat.json",
+      ].map(path.normalize),
+      proceedLabel: InstallChoices.Proceed,
+      proceedOutInstructions: [
+        {
+          type: "copy",
+          source: path.join(ARCHIVE_PREFIX, "proximas_propshop_v4.archive"),
+          destination: path.join(ARCHIVE_PREFIX, "proximas_propshop_v4.archive"),
+        },
+        {
+          type: "copy",
+          source: path.normalize(
```

Comment:
I added a couple helpers for the expected instructions: https://github.com/E1337Kat/cyberpunk2077_ext_redux/blob/a7f5728c7b10dc43dd91f50206947ad350604eb2/test/unit/utils.helper.ts#L41

Reply:
Thanks, I hadn't noticed that :)